src/Arcus.WebApi.sln

```diff
@@ -17,7 +17,16 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Arcus.WebApi.OpenApi.Extens
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Arcus.WebApi.Security.Authentication", "Arcus.WebApi.Security.Authentication\Arcus.WebApi.Security.Authentication.csproj", "{88EADCAB-29BF-42F1-8748-646676898903}"
 EndProject
+<<<<<<< HEAD
```

Comment:
This doesn't look good :D

Reply:
😄 yeah, but isn't that the outdated version?

Reply:
Nevermind me 🙄